samples/sensor/wsen_hids/prj.conf

```diff
@@ -0,0 +1,16 @@
+###### Basic configuration required for using sensors ######
+CONFIG_I2C=y
+# CONFIG_SPI=y
```

Comment:
probably nice to comment why SPI is disabled, probably can encourage user to have a look at relevate file

Reply:
I simply removed the line.